mani_skill/envs/tasks/drawing/draw_triangle.py

```diff
@@ -0,0 +1,330 @@
+import math
+import random
+from typing import Dict
+
+import mani_skill.envs.utils.randomization as randomization
+import numpy as np
+import sapien
+import torch
+from mani_skill.agents.robots.panda.panda_stick import PandaStick
+from mani_skill.envs.sapien_env import BaseEnv
+from mani_skill.sensors.camera import CameraConfig
+from mani_skill.utils import sapien_utils
+from mani_skill.utils.geometry.rotation_conversions import quaternion_to_matrix
+from mani_skill.utils.registration import register_env
+from mani_skill.utils.scene_builder.table.scene_builder import \
+    TableSceneBuilder
+from mani_skill.utils.structs.actor import Actor
+from mani_skill.utils.structs.pose import Pose
+from mani_skill.utils.structs.types import SceneConfig, SimConfig
+from transforms3d.euler import euler2quat, quat2euler
+from transforms3d.quaternions import quat2mat
+
+
+@register_env("DrawTriangle-v1", max_episode_steps=1000)
+class DrawTriangle(BaseEnv):
+    """
+    This is a simple environment demonstrating drawing simulation on a table with a robot arm. There are no success/rewards defined, users can use this code as a starting point
+    to create their own drawing type tasks.
+    """
+
+    MAX_DOTS = 1010
```

Comment:
reduce this down significantly, maybe to 300. and max episode steps should be down to 300